TcgPocket/Features/Games/Game.cs

```diff
@@ -10,6 +12,7 @@ public class Game : GameGetDto, IEntity
     public List<CardType> CardTypes { get; set; }
     public List<Attributes.Attribute> Attributes { get; set; }
```

Comment:
Do this in conjunction with the above comment. I didn't catch this on Zoe's PR.. mb

```suggestion
    public List<Attribute> Attributes { get; set; }
```